invenio_app_ils/ui/src/invenio_app_ils/common/api/loan.js

```diff
@@ -36,6 +36,41 @@ const postAction = (
   return http.post(url, payload);
 };
 
+const buildPendingQuery = (documentPid, itemPid) => {
+  const qsDoc = documentPid ? `document_pid:${documentPid}` : '';
+  const qsItem = itemPid ? `item_pid:${itemPid}` : '';
+  const qsDocItem =
+    qsDoc && qsItem ? `(${qsDoc} OR ${qsItem})` : `${qsDoc}${qsItem}`;
+  return `${qsDocItem} AND state:PENDING`;
+};
+
+const assignItemToLoan = (itemPid, loanPid) => {
+  const newItemRef = {
+    $ref: `https://127.0.0.1:5000/api/resolver/circulation/items/${itemPid}`,
```

Comment:
@topless Can you create beforehand the ticket to address this after?

Reply:
Issue #112